tests/unit/moduleapi/list.tcl

```diff
@@ -63,4 +63,24 @@ start_server {tags {"modules"}} {
         r list.edit k reverse rkr foo bar
         r list.getall k
     } {bar y foo}
+
+    proc sleep {time} {
+        after $time set end 1
+        vwait end
+    }
+
+    test {Busy module} {
+        set rd [redis_deferring_client]
+        $rd list.busy
```

Comment:
let's set `script-time-limit` to a low number, so that this test will not take long